coalib/bearlib/abstractions/Linter.py

```diff
@@ -516,9 +528,12 @@ def process_output_regex(
                 An iterator returning results.
             """
             for match in re.finditer(output_regex, output):
-                yield self._convert_output_regex_match_to_result(
-                    match, filename, severity_map=severity_map,
-                    result_message=result_message)
+                try:
```

Comment:
We want the `Result` to be emitted, just without the line number.
c.f. https://github.com/coala/coala/issues/5600#issuecomment-403055446 and sorry for the confusion.